doc/classes/EditorSettings.xml

```diff
@@ -219,6 +222,11 @@
 		<member name="docks/property_editor/subresource_hue_tint" type="float" setter="" getter="">
 			The tint intensity to use for the subresources background in the Inspector dock. The tint is used to distinguish between different subresources in the inspector. Higher values result in a more noticeable background color difference.
 		</member>
+		<member name="docks/scene_tree/ask_before_deleting_related_animation_tracks" type="bool" setter="" getter="">
+			If [code]true[/code], when a node is deleted with animation tracks referencing it, a confirmation dialog appears before the tracks are deleted. The dialog will appear even when using the "Delete (No Confirm)" shortcut.
+		</member>
+		<member name="docks/scene_tree/ask_before_revoking_unique_name" type="bool" setter="" getter="">
+		</member>
```

Comment:
```suggestion
		<member name="docks/scene_tree/ask_before_revoking_unique_name" type="bool" setter="" getter="">
			If [code]true[/code], displays a confirmation dialog before clicking the "percent" icon next to a node name in the Scene tree dock. When clicked, this icon revokes the node's scene-unique name, which can impact the behavior of scripts that rely on this scene-unique name.
		</member>
```

For reference, this setting was added by https://github.com/godotengine/godot/pull/95343.